sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -68,7 +68,9 @@ class AdaptiveQueryExecSuite
     val result = dfAdaptive.collect()
     withSQLConf(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "false") {
       val df = sql(query)
-      QueryTest.sameRows(result.toSeq, df.collect().toSeq)
+      QueryTest.sameRows(result.toSeq, df.collect().toSeq).foreach {
```

Comment:
good catch! can we use `checkAnswer(df, result)`?

Reply:
Yes. I will update it